stories/Button.stories.ts

```diff
@@ -16,10 +16,10 @@ const meta = {
       control: { type: 'radio' },
     },
   },
-} satisfies Meta<typeof Button>;
```

Comment:
Maybe just keep the `satisfies` for the autocomplete/type checking of the meta object!

Reply:
kept it, moved to line 5